_data/library.yml

```diff
@@ -648,71 +648,6 @@
     clusters, managing Helm, and generating and securely storing TLS certificates.
 
-- name: "Consul"
-  icons:
-    - title: Terraform
-      image: icon-terraform.png
-    - title: Bash
-      image: icon-bash.png
-  public_url: "https://github.com/hashicorp/terraform-aws-consul"
-  type: Open Source
-  description: |
-    Deploy a best-practices HashiCorp Consul cluster. Includes support for automatic bootstrapping, configuring dnsmasq
-    to use Consul as a DNS server, access to the Consul UI, and automatic recovery of failed servers.
-  submodules:
-    - name: install-consul
-      blurb: install Consul on a Linux server
-      description: Install Consul and its dependencies on a Linux server.
-    - name: install-dnsmasq
-      blurb: install dnsmasq on a Linux server
-      description: Install dnsmasq on a Linux server and configure it to work with Consul as a DNS server. This allows you to use domain names such as my-app.service.consul.
-    - name: run-consul
-      blurb: configure and run Consul
-      description: Run Consul and automatically bootstrap the cluster.
-
-- name: "Nomad"
-  icons:
-    - title: Terraform
-      image: icon-terraform.png
-    - title: Bash
-      image: icon-bash.png
-  public_url: "https://github.com/hashicorp/terraform-aws-nomad"
-  type: Open Source
-  description: |
-    Deploy a best-practices HashiCorp Nomad cluster. Includes support for automatic bootstrapping, automatic discovery
-    of Consul servers (for cluster coordination), automatic recovery of failed servers.
-  submodules:
-    - name: install-nomad
-      blurb: install Nomad on a Linux server
-      description: Install Nomad and its dependencies on a Linux server.
-    - name: run-nomad
-      blurb: configure and run Nomad
-      description: Run Nomad and automatically connect to a Consul cluster.
-
-- name: "Vault"
-  icons:
-    - title: Terraform
-      image: icon-terraform.png
-    - title: Bash
-      image: icon-bash.png
-  public_url: "https://github.com/hashicorp/terraform-aws-vault"
-  type: Open Source
-  description: |
-    Deploy a best-practices HashiCorp Vault cluster for secrets management. Includes support for automatic
-    bootstrapping, automatic discovery of Consul clusters (for HA cluster coordination), using S3 as a storage backend,
-    creating self-signed TLS certificates, updating the OS certificate store, configuring an ELB in front of Vault to
-    allow public access, and automatic recovery of failed servers.
-  submodules:
-    - name: install-vault
-      blurb: install Vault on a Linux server
-      description: Install Vault and its dependencies on a Linux server.
-    - name: run-vault
-      blurb: configure and run Vault
-      description: Run Vault and automatically connect to a Consul cluster.
-    - name: private-tls-cert
-      blurb: generate a self-signed TLS cert
-      description: Generate self-signed TLS certificates for use with Vault.
-
 - name: "Couchbase"
```

Comment:
Couchbase can also be removed because our contract expired.

Reply:
Thanks, pushed a new commit to remove Couchbase as well.